src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -153,7 +153,8 @@ private static synchronized int generateDebugId()
     /**
      * Data channels mapped by SCTP stream identified(sid).
      */
-    private final Map<Integer,WebRtcDataStream> channels = new HashMap<>();
+    private final Map<Integer,WebRtcDataStream> channels
+        = new ConcurrentHashMap<>();
```

Comment:
I see in the source code of SctpConnection that the access to the field channels is synchronized on the field syncRoot. Did you see an inconsistency that prompted the necessity of ConcurrentHashMap?